Update chrome driver link to fix ;data; problem

CreativiTimothy opened the pull request:
Lots of people said it wasn't working because the browser automatically closed after ";data;". I updated the link to reflect the latest Chrome Driver which fixes that problem

CreativiTimothy committed:
Update chrome driver link to fix data problem

JapuDCret commented:
thank you, this fixes the browser closing...

but it seems that due to changes there is an error while verifying if the account was successfully created..

`Making account #1
Attempting to create user wingskoyplaw:sleeper89. Opening browser...
Step 1: Verifying age using birthday: 1989-12-17
Step 2: Entering account details
You did not pass a 2captcha key. Please solve the captcha manually.
Captcha successful. Sleeping for 1 second...
Error StaleElementReferenceException!
Failed to create user: wingskoyplaw
Attempting to create user wingskoyplaw:sleeper89. Opening browser...`

CreativiTimothy committed:
Updated program files to 86x

downloaded latest google chrome version and it appears to be installed to progarm files (x86) by default

CreativiTimothy commented:
@JapuDCret Yeah. I get it a couple times, sometimes saying the user already exists, and sometimes failing to verify email and TOS. Not sure how to fix it though

JapuDCret commented:
ok i got it now.
in Issue #65 kamaze explains why this error occurs:

> @thedeathbox it seems PTC now checking the email format.
> using random email would get "Failed to create user" error.
> after using real email address, then it works now.
> (i'm using 2captcha, it still can work for now)

By Using the Plusmail Trick and the gmail Auto Verification already built in, you can easily fix this problem, for more info please refer to: https://github.com/sriyegna/Pikaptcha#using-auto-email-verify
